<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tracker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QNavBar" name="navbar"/>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QListView" name="listView"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTreeView" name="treeView"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_3">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QColumnView" name="columnView"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Window"/>
    <addaction name="actionNew_Tab"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Folder"/>
    <addaction name="actionNew_Smart_Folder"/>
    <addaction name="actionNew_Burning_Folder"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_With"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionGet_Information"/>
    <addaction name="separator"/>
    <addaction name="actionCompress"/>
    <addaction name="separator"/>
    <addaction name="actionDuplicate"/>
    <addaction name="actionCreate_Alias"/>
    <addaction name="separator"/>
    <addaction name="actionMove_to_Trash"/>
    <addaction name="actionEject"/>
    <addaction name="actionBurn"/>
    <addaction name="separator"/>
    <addaction name="actionSearch"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionSelect_All"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionAs_Icons"/>
    <addaction name="actionAs_List"/>
    <addaction name="actionAs_Columns"/>
    <addaction name="separator"/>
    <addaction name="actionRearrange"/>
   </widget>
   <widget class="QMenu" name="menuGo">
    <property name="title">
     <string>Go</string>
    </property>
    <addaction name="actionBack"/>
    <addaction name="actionForward"/>
    <addaction name="actionFolder"/>
    <addaction name="separator"/>
    <addaction name="actionComputer"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuGo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Window">
   <property name="text">
    <string>New Window</string>
   </property>
   <property name="shortcut">
    <string>Meta+N</string>
   </property>
  </action>
  <action name="actionNew_Tab">
   <property name="text">
    <string>New Tab</string>
   </property>
   <property name="shortcut">
    <string>Meta+T</string>
   </property>
  </action>
  <action name="actionNew_Folder">
   <property name="text">
    <string>New Folder</string>
   </property>
  </action>
  <action name="actionNew_Smart_Folder">
   <property name="text">
    <string>New Smart Folder</string>
   </property>
  </action>
  <action name="actionNew_Burning_Folder">
   <property name="text">
    <string>New Burning Folder</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Meta+O</string>
   </property>
  </action>
  <action name="actionOpen_With">
   <property name="text">
    <string>Open With</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionGet_Information">
   <property name="text">
    <string>Get Information</string>
   </property>
  </action>
  <action name="actionCompress">
   <property name="text">
    <string>Compress</string>
   </property>
  </action>
  <action name="actionDuplicate">
   <property name="text">
    <string>Duplicate</string>
   </property>
  </action>
  <action name="actionCreate_Alias">
   <property name="text">
    <string>Create Alias</string>
   </property>
  </action>
  <action name="actionMove_to_Trash">
   <property name="text">
    <string>Move to Trash</string>
   </property>
  </action>
  <action name="actionEject">
   <property name="text">
    <string>Eject</string>
   </property>
  </action>
  <action name="actionBurn">
   <property name="text">
    <string>Burn...</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionAs_Icons">
   <property name="text">
    <string>As Icons</string>
   </property>
  </action>
  <action name="actionAs_List">
   <property name="text">
    <string>As List</string>
   </property>
  </action>
  <action name="actionAs_Columns">
   <property name="text">
    <string>As Columns</string>
   </property>
  </action>
  <action name="actionRearrange">
   <property name="text">
    <string>Rearrange</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="text">
    <string>Forward</string>
   </property>
  </action>
  <action name="actionFolder">
   <property name="text">
    <string>Folder</string>
   </property>
  </action>
  <action name="actionComputer">
   <property name="text">
    <string>Computer</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QNavBar</class>
   <extends>QWidget</extends>
   <header>QubeGui/QNavBar</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
